### PR TITLE
Add caneton stats endpoint

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: tempo_apio.py [hunk 1]
```diff
@@ -44,5 +44,11 @@ def get_articles():
                     mimetype="application/json")
 
 
+@app.route('/caneton_stats')
+def get_caneton_stats():
+    s = db.stats.find({}, {'_id': False}).sort('datetime', pymongo.DESCENDING)
+    return jsonify(list(s))
```
Comment (samueldg) [inline tempo_apio.py:50]: Doesn't that break with the ObjectId and bson-specific stuff? Maybe use the bson.json_utils that's used for articles to be sure?

File: tempo_apio.py [hunk 2]
```diff
@@ -44,5 +44,11 @@ def get_articles():
                     mimetype="application/json")
 
 
+@app.route('/caneton_stats')
+def get_caneton_stats():
+    s = db.stats.find({}, {'_id': False}).sort('datetime', pymongo.DESCENDING)
```
Comment (samueldg) [inline tempo_apio.py:49]: Curious here, what's the purpose of the first vs. second params to find?

File: tempo_apio.py (hunk 1 above)
Comment (ghost) [reply tempo_apio.py:50]: http://stackoverflow.com/a/12345646/5732783

I tested this and it removes the id completely. The response of the api endpoint on my instance:

```json
[
  {
    "articles_added": 0, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:30:56.027322", 
    "obamo_json_errors": 0
  }, 
  {
    "articles_added": 0, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:25:55.064296", 
    "obamo_json_errors": 0
  }, 
  {
    "articles_added": 0, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:21:00.769815", 
    "obamo_json_errors": 0
  }, 
  {
    "articles_added": 1, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:16:01.428559", 
    "obamo_json_errors": 0
  }, 
  {
    "articles_added": 0, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:11:31.399178", 
    "obamo_json_errors": 1
  }, 
  {
    "articles_added": 0, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:06:06.391945", 
    "obamo_json_errors": 1
  }, 
  {
    "articles_added": 0, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:00:36.214950", 
    "obamo_json_errors": 1
  }, 
  {
    "articles_added": 18, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 03:56:22.269037"
  }, 
  {
    "articles_added": 0, 
    "datetime": "2017-03-26 03:04:13.907514"
  }, 
  {
    "articles_added": 0, 
    "datetime": "2017-03-26 02:52:28.782933"
  }, 
  {
    "articles_added": 0, 
    "datetime": "2017-03-26 02:41:55.538538"
  }, 
  {
    "articles_added": 0, 
    "datetime": "2017-03-26 01:50:13.340902"
  }
]
```

Comment (samueldg) [reply tempo_apio.py:50]: Ah interesting! So it addressed both comments then!

Comment (ghost) [reply tempo_apio.py:50]:
```python
>>> list(db.stats.find())
[{'_id': ObjectId('58d71e55a40f530005bf8573'), 'datetime': '2017-03-26 01:50:13.340902', 'articles_added': 0}, {'_id': ObjectId('58d72a73747c9800062098f4'), 'datetime': '2017-03-26 02:41:55.538538', 'articles_added': 0}, {'_id': ObjectId('58d72cec747c9800062098f5'), 'datetime': '2017-03-26 02:52:28.782933', 'articles_added': 0}, {'_id': ObjectId('58d72fad595ba500062038f6'), 'datetime': '2017-03-26 03:04:13.907514', 'articles_added': 0}, {'_id': ObjectId('58d73be6b7c6f20005498761'), 'datetime': '2017-03-26 03:56:22.269037', 'articles_added': 18, 'articles_fetched': 55}, {'_id': ObjectId('58d73ce4422c020005e281a8'), 'datetime': '2017-03-26 04:00:36.214950', 'articles_added': 0, 'articles_fetched': 55, 'obamo_json_errors': 1}, {'_id': ObjectId('58d73e2e422c020005e281a9'), 'datetime': '2017-03-26 04:06:06.391945', 'articles_added': 0, 'articles_fetched': 55, 'obamo_json_errors': 1}, {'_id': ObjectId('58d73f73422c020005e281aa'), 'datetime': '2017-03-26 04:11:31.399178', 'articles_added': 0, 'articles_fetched': 55, 'obamo_json_errors': 1}, {'_id': ObjectId('58d74081422c020005e281ac'), 'datetime': '2017-03-26 04:16:01.428559', 'articles_added': 1, 'articles_fetched': 55, 'obamo_json_errors': 0}, {'_id': ObjectId('58d741ac422c020005e281ad'), 'datetime': '2017-03-26 04:21:00.769815', 'articles_added': 0, 'articles_fetched': 55, 'obamo_json_errors': 0}, {'_id': ObjectId('58d742d3422c020005e281ae'), 'datetime': '2017-03-26 04:25:55.064296', 'articles_added': 0, 'articles_fetched': 55, 'obamo_json_errors': 0}, {'_id': ObjectId('58d74400422c020005e281af'), 'datetime': '2017-03-26 04:30:56.027322', 'articles_added': 0, 'articles_fetched': 55, 'obamo_json_errors': 0}]
>>> list(db.stats.find({}, {'_id': False}))
[{'datetime': '2017-03-26 01:50:13.340902', 'articles_added': 0}, {'datetime': '2017-03-26 02:41:55.538538', 'articles_added': 0}, {'datetime': '2017-03-26 02:52:28.782933', 'articles_added': 0}, {'datetime': '2017-03-26 03:04:13.907514', 'articles_added': 0}, {'datetime': '2017-03-26 03:56:22.269037', 'articles_added': 18, 'articles_fetched': 55}, {'datetime': '2017-03-26 04:00:36.214950', 'articles_added': 0, 'articles_fetched': 55, 'obamo_json_errors': 1}, {'datetime': '2017-03-26 04:06:06.391945', 'articles_added': 0, 'articles_fetched': 55, 'obamo_json_errors': 1}, {'datetime': '2017-03-26 04:11:31.399178', 'articles_added': 0, 'articles_fetched': 55, 'obamo_json_errors': 1}, {'datetime': '2017-03-26 04:16:01.428559', 'articles_added': 1, 'articles_fetched': 55, 'obamo_json_errors': 0}, {'datetime': '2017-03-26 04:21:00.769815', 'articles_added': 0, 'articles_fetched': 55, 'obamo_json_errors': 0}, {'datetime': '2017-03-26 04:25:55.064296', 'articles_added': 0, 'articles_fetched': 55, 'obamo_json_errors': 0}, {'datetime': '2017-03-26 04:30:56.027322', 'articles_added': 0, 'articles_fetched': 55, 'obamo_json_errors': 0}]
```

**Pull request merged**